### DISCUSSION TITLE
How to send messages periodically?

### DISCUSSION THREAD

**Discussion by sajjadsabzkar** [aiogram/aiogram#858] [Q&A] [answered]
Hi , thanks for developing aiogram .
Suppose I want to build a robot that sends a message periodically (every hour / every day / every week) . i don't see example in your document or google search , i hope  there is a solution

**Comment by MassonNN** [comment, marked as answer]
so, you can do this by using [APScheduler](https://apscheduler.readthedocs.io/en/stable/userguide.html)

**Comment by sajjadsabzkar** [comment]
> so, you can do this by using [APScheduler](https://apscheduler.readthedocs.io/en/stable/userguide.html)

thanks you
